funcx_endpoint/funcx_endpoint/cli.py

```diff
@@ -60,7 +60,7 @@ def common_options(f):
         expose_value=False,
         callback=log_flag_callback,
         is_eager=True,
-        help="Emit extra information generally only helpful for developers",
+        help="Emit extra logging information to console, helpful for developers and others in easier visibility in endpoint activities",
```

Comment:
Actually, the `--debug` flag only determines whether debug-level logging is enabled.  It's the new flag, `--log-to-console`, that determines if logs are sent to the console.  Maybe more simply:

> Emit extra logging information.  (Warning: very verbose)

It's named `--debug`, so for almost all folks who might be interested in that flag, the contextual clue as to what it does is probably enough.  Yeah?

Reply:
Oops, that was as mis-reading of the previous PR.  For some reason thought the help message was also for the log to console change.  Will delete as it isn't necessary for your PR (debug option is pre-existing)